File: TinyScript/java/src/main/java/lexer/TokenType.java
package lexer;

/**
 * @author MFine
 * @version 1.0
 * @date 2021/6/29 0:35
 **/
public enum TokenType {
    // 关键字
    KEYWORD,
    // 变量
    VARIABLE,
    // 操作符
    OPERATOR,
    // 括号
    BRACKET,
    // 字符串
    STRING,
    // 浮点数
    FLOAT,
    // 布尔型
    BOOLEAN,
    // 整型
    INTEGER
}

File: TinyScript/js/src/main/js/lexer/TokenType.ts
/*
 * @Author: MFine
 * @Date: 2021-06-29 00:49:31
 * @LastEditTime: 2021-06-29 01:08:43
 * @LastEditors: MFine
 * @Description:
 */
enum TokenType {
    // 关键字
    KEYWORD = 0,
    // 变量
    VARIABLE = 1,
    // 操作符
    OPERATOR = 2,
    // 括号
    BRACKET = 3,
    // 整型
    INTEGER = 4,
    // 浮点数
    FLOAT = 5,
    // 布尔型
    BOOLEAN = 6,
    // 字符串
    STRING = 7,
}

export default TokenType;
